### PR TITLE
chore(naming): use `docsearch` everywhere

### LINKED ISSUE #40: docsearch or docSearch
our `<script>` build is exposing `docsearch()` and npm is exposing `docSearch`. (currently the documentation wrongly uses `docSearch`)

### PR DESCRIPTION
**Pull request by redox**
Fix #40

### CODE REVIEW

**Comment by pixelastic** [comment]
You have a weird diff in the readme.

**Comment by redox** [comment]
> You have a weird diff in the readme.

Yes it's because I regenerated the `documentation.md` to take into account my update but it seems it was kind of outdated already.

File: docs/documentation.md [hunk 1]
```diff
@@ -6,15 +6,15 @@ permalink: /documentation/
 
 ## Introduction
 
-We're scratching our own itch here. As developers, we spend a lot of time reading documentation, and it isn’t always easy to find the information we need.
+We're scratching our own itch here. As developers, we spend a lot of time reading documentation, and it isn't always easy to find the information we need.
 
 Not blaming anyone here. Building a good search for a documentation is a complex challenge. We happen to have a lot of experience doing that, and we want to share it with the world. For free.
 
-Just submit the form on the [community.algolia.com/docsearch/](https://community.algolia.com/docsearch/) website and we'll get back to you with what you need to integrate your new search into your website.
+Just submit the form on the [website][12] and we'll get back to you with what you need to integrate your new search into your website.
```
Comment (redox) [inline docs/documentation.md]: Oh that link is not gonna work btw :/

**Comment by redox** [comment]
I've fixed the links + regen the `documentation.md`

**Pull request merged**